scratch/settings.py

```diff
@@ -163,3 +163,7 @@
 FIXTURE_DIRS = (
    '/app/fixtures/',
 )
+
+EMAIL_USE_TLS = False
```

Comment:
For EMAIL_USE_TLS take environment variable as boolean as for DEBUG:
      
          DEBUG = env('DEBUG', False)

For EMAIL_HOST_USER AND EMAIL_HOST_PASSWORD, as string:

        VARIABLE = env('VARIABLE', '')